packages/record/src/record/utils/provide-cookie-access.ts

```diff
@@ -0,0 +1,43 @@
+import { Cookie } from "@alwaysmeticulous/api";
+import { Page, Protocol } from "puppeteer";
+
+type ModifiedWindow = {
+  __meticulous?: {
+    getAllCookies?: () => Promise<Cookie[]>;
+  };
+};
+
+export const provideCookieAccess = async (page: Page) => {
+  const client = await page.target().createCDPSession();
+  const cookies = (await client.send("Network.getAllCookies")).cookies.map(
```

Comment:
Does this differ from `Page.cookies()`?

Reply:
I beleive that one doesn't include http only cookies https://stackoverflow.com/questions/49389775/puppeteers-page-cookies-not-retrieving-all-cookies-shown-in-the-chrome-dev-to